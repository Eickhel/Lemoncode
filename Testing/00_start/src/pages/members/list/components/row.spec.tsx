import { shallow } from "enzyme";
import * as React from "react";
import { Row } from "./row";

describe("pages/members/list/components/row tests", () => {
  it("should render as expected when passing required properties", () => {
    // Arrange
    const props = {
      member: {
        id: 14540103,
        name: "antonio06",
        avatarUrl: "https://avatars1.githubusercontent.com/u/14540103?v=4"
      }
    };

    // Act
    const component = shallow(<Row {...props} />);

    // Assert
    expect(component).toMatchSnapshot();
  });
});
