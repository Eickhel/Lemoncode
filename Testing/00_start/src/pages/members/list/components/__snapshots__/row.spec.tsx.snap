// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`pages/members/list/components/row tests should render as expected when passing required properties 1`] = `
<tr>
  <td>
    <img
      src="https://avatars1.githubusercontent.com/u/14540103?v=4"
      style={
        Object {
          "maxWidth": "10rem",
        }
      }
    />
  </td>
  <td>
    <span>
      14540103
    </span>
  </td>
  <td>
    <span>
      antonio06
    </span>
  </td>
</tr>
`;
